### DISCUSSION TITLE
yt-dlp

### DISCUSSION THREAD

**Discussion by elverlin** [flet-dev/flet#4184] [Packages] [answered]
Requests for adding yt-dlp pacakge.

**Comment by hieulbp** [comment]
hope i can use yt-dlp

**Comment by ndonkoHenri** [comment, marked as answer]
yt-dlp is pure-Python and works out of the box. Just declare it in your project dependencies. I tested it myself on Android and iOS, and it works. 

Let know if you face any issues.

**Reply by hieulbp** [reply]
Can you send me a sample project?
